Browse table bookmark form: Clickable labels

mynetx opened the pull request:
Make text field and check box labels clickable in the Browse table page,
create SQL bookmark form.

lem9 commented:
Hi,
we are trying to clarify whether or not we can accept contributions without a person name; meanwhile, is it possible for you to redo this commit, using a full name, like we show in https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO ?

Commit:
Browse table bookmark form: Clickable labels

Make text field and check box labels clickable in the Browse table page,
create SQL bookmark form.

Signed-off-by: Joshua Atkins <support@jublo.net>

jublo commented:
@lem9 lmk about the outcome of your clarification.

lem9 commented:
Thanks!
